modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/Select/BaseSelect.tsx

```diff
@@ -73,19 +91,57 @@ export function BaseSelect({
 			required={required}
 		>
 			<ClayAutocomplete>
-				<ClayAutocomplete.Input
-					defaultValue={value}
-					disabled={disabled}
-					onClick={() => setDropdownActive((active) => !active)}
-					placeholder={placeholder}
-					ref={inputRef}
-					value={value}
-					{...restProps}
-				/>
+				{useMultiSelect ? (
+					<ClayMultiSelect
+						disabled={disabled}
+						items={multiSelectItems as MultiSelectItem[]}
+						onClick={() => setDropdownActive((active) => !active)}
+						onInput={() => {}}
+						onItemsChange={(items: MultiSelectItem[]) => {
```

Comment:
@danvb112 We also need to make sure that when the user clicks on the "X" button to remove all the options selected, the Select All option is unchecked too.

Reply:
Done! Thanks for noticing it @naraandrade